Justfile

```diff
@@ -70,3 +70,7 @@ build-docs:
 # Deploy docs to Github Pages.
 publish-docs:
     poetry run mkdocs gh-deploy --force
+
+# Run the fixie CLI.
+cli *FLAGS:
```

Comment:
No need for this. We can do `poetry run fixie ...` in this repo and it picks the latest.

Reply:
how about I make this `just fixie` which does what you say above?

Reply:
sure that'd be easier to type